modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -8129,6 +8130,7 @@ limit=Limit
 limit-membership-to-members-of-the-parent-site=Limit membership to members of the parent site
 limit-organization=Limit Organization
 limit-scope=Limit Scope
+limit-to-one-submission-per-user=Limit to one submission per user
```

Comment:
@marcelabc same as above.

Reply:
There is no period in the mockup 😕
<img width="692" alt="Screen Shot 2021-11-03 at 11 57 17" src="https://user-images.githubusercontent.com/15161924/140084936-077464bc-b216-46fe-b3b8-4945f0f7c67e.png">

Reply:
Maybe Brian will change, but let's keep it this way because of the mockup.
Thanks @marcelabc